tests/python/mkl/test_quantization_cpu.py

```diff
@@ -0,0 +1,309 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import mxnet as mx
+import numpy as np
+from mxnet.test_utils import assert_almost_equal, rand_ndarray, rand_shape_nd, same, set_default_context
+
+
+def test_quantize_float32_to_int8():

```

Comment:
Can you not duplicate test code? You can import required test functions from test_quantization.py for tests that can run w/ or w/o mkldnn.

For the CPU test that can only run with MKLDNN and cuDNN, it's okay to duplicate code for now but please add necessary comments about where the test code comes from. In the future, we want to remove the duplicate code.

Reply:
Currently the mainly difference is MKLDNN use uint8 and GPU use int8, and we don't support fc int8 right now, we will try to share code with GPU.